bottle.py

```diff
@@ -952,7 +952,7 @@ def wrapper(handler):
         return wrapper
 
     def default_error_handler(self, res):
-        return tob(template(ERROR_PAGE_TEMPLATE, e=res))
+        return tob(template(ERROR_PAGE_TEMPLATE, e=res, template_settings=dict(name='__ERROR_PAGE_TEMPLATE')))
```

Comment:
Open to a better name here. Should be something that's clearly the default template handler, but not going to clash with any names in userspace